packages/strapi-parts/src/Carousel/Carousel.stories.mdx

```diff
@@ -21,7 +22,20 @@ This is the doc of the `Carousel` component
 
 ## Carousel
 
-Description...
+Carousels are slideshow components for navigating through a set of assets. 
+
+**Best practices**
+
+- Carousels should be mostly used in forms.
+- Carousels can also be used to display only one asset.
+- The end of a Carousel label should indicate the amount of included assets.
+- A set of actions is possible via Icon Buttons within a Carousel.
+- Carousels should at least give the possibility to remove an asset from it.
```

Comment:
Same here, I don't really understand why we need to force a use case on the CTAs of that component?

Reply:
The Carousel component is the one we call "Media field" within the CTB, it's different from the Card component that we use within the Media Library, hence the proposal. However, this statement is not always true and I agree with that (permissions, ...) :)